### PR TITLE
Logging of the vm_server

### PR DESCRIPTION
**Pull request by NKH123**
Added debug statements in the vm_server.
The logs are saved in server.log file.
Potentially useful information such as the contents of the proto file, accepted request and the time at which the request arrived etc. are stored in server.log
Also changed the way output files are being handled, now they are all moved to the specified directory that is described in the request proto.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -135,4 +135,7 @@ dmypy.json
 .pytype/
 
 # Cython debug symbols
-cython_debug/
\ No newline at end of file
+cython_debug/
+
+#generated proto binary files
+*.pb
```
Comment (paneendra-google) [inline .gitignore:141]: add newline at EOF

File: vm_server/accept/server.py [hunk 2]
```diff
@@ -18,28 +19,72 @@
...
+  """
+  logging.debug("Removing execute directory")
   dirpath = Path("..\\execute")
-  if dirpath.exists() and dirpath.is_dir(): # delete leftover files
-    shutil.rmtree(dirpath)
+  try: 
+    if dirpath.exists() and dirpath.is_dir(): # delete leftover files
+      shutil.rmtree(dirpath)
+  except Exception as exception:  #catch errors if any
+    print(exception)
```
Comment (paneendra-google) [inline vm_server/accept/server.py:34]: can remove print since you are logging, also same for all other print statements

File: .gitignore (hunk 1 above)
Comment (NKH123) [reply .gitignore]: Added newline

File: vm_server/accept/server.py (hunk 2 above)
Comment (NKH123) [reply vm_server/accept/server.py]: REmoved print statement

File: vm_server/accept/server.py [hunk 3]
```diff
@@ -18,28 +19,68 @@
...
+    task_response: an object of TaskResponse() that will be sent back
+  """
+  logging.debug("Removing execute directory")
   dirpath = Path("..\\execute")
-  if dirpath.exists() and dirpath.is_dir(): # delete leftover files
-    shutil.rmtree(dirpath)
+  try:
+    if dirpath.exists() and dirpath.is_dir(): # delete leftover files
+      shutil.rmtree(dirpath)
+  except Exception as exception:  #catch errors if any
```
Comment (kaladin-google) [inline vm_server/accept/server.py:33]: This is probably the minutest of nit picks that I have ever done. (critique automatically points this out) - extra space before comment.

Comment (NKH123) [reply vm_server/accept/server.py]: Added spaces after hash :)

**Pull request merged**